332 vesting UI

gala1234 opened the pull request:
# Description

Closes: https://github.com/nymtech/nym/issues/2339

gala1234 committed:
fixing dark mode progress bar

gala1234 committed:
fixing dark mode progress bar

gala1234 committed:
fixing dark mode progress bar

gala1234 committed:
fixing row color, and using correct green

gala1234 committed:
fixing row color, and using correct green

gala1234 committed:
fixing row color, and using correct green

doums commented on nym-wallet/src/pages/balance/components/vesting-timeline.tsx:
```diff
@@ -30,18 +34,30 @@ export const VestingTimeline: React.FC<{ percentageComplete: number }> = ({ perc
+              +percentageComplete.toFixed(2) >= calculateMarkerPosition(arr.length, i)
+                ? mode === 'light'
+                  ? '#121726'
+                  : '#21D073'
```
Hi, these color values exist in the wallet palette, `text.dark` for `#121726`, `success` for `#21D073`, should be better to access them from the theme instead of using hard coded values.

neacsu committed:
Fix NC filter for domains suffix-only domains (#1487)

* Fix NC filter for domains suffix-only domains

* Update CHANGELOG

* Fix unit test for filter

Some domains might be composed of the suffix only.

There are no nonsense domains, as they can be defined even on the local
machine. The underlying library doesn't resolve them, but rather uses a
fixed list of public suffixes to assess the domains.

* Fix clippy

gala1234 committed:
Merge branch '332-vesting-ui' of github.com:nymtech/nym into 332-vesting-ui

gala1234 committed:
remove hardcoded colours

gala1234 committed:
Merge branch '332-vesting-ui' of github.com:nymtech/nym into 332-vesting-ui

gala1234 commented:
After rebase with `develop`, I don't know why, was appearing here a Bodgan commit, I've been trying to fix it, but finally I saw easier to create a new branch from develop and apply on this one the changes again.

Here is the new PR: https://github.com/nymtech/nym/pull/1489
